Holovibes/sources/compute/converts.cc

```diff
@@ -118,7 +118,7 @@ void Converts::insert_to_modulus_moments(float* output, const ushort f_start, co
             complex_to_modulus_moments(output,
                                        time_transformation_env_.gpu_p_acc_buffer,
                                        fd_.get_frame_res(),
-                                       f_start,
+                                       f_start - 1,
```

Comment:
for all -1 you add it will be maybe better to remove it and put the -1 at the init of moments_env_.f_start in pipe.cc line 397. with a comment to explain why.